persistence/sql/migrations/sql/2022080818000000000_identity_credentials_types_saml.up.sql

```diff
@@ -0,0 +1 @@
+INSERT INTO identity_credential_types (id, name) SELECT 'ff5a1823-8b47-4255-860f-4b70ed122740', 'saml' WHERE NOT EXISTS ( SELECT * FROM identity_credential_types WHERE name = 'saml');
```

Comment:
How did you generate this UUID? :)

Reply:
I generated it by hand because I still didn't understand how it all worked, how am I supposed to do that?